src/classes/RecipeRepository.js

```diff
@@ -38,26 +38,26 @@ class RecipeRepository {
     return ingredientIDFromSearch;
   };
 
-  filterByIng(nameOrIng, ingredientsData) {
+  filterByIng(nameOrIng, ingredientsData, recipes) {
```

Comment:
Adding this recipes parameter (an array) makes this more dynamic. Before we could only filter from the ENTIRE data set but now we can to filter other arrays as well- what's in the cookbook and what's been favorited.